pkgs/by-name/pc/pcsx2/package.nix

```diff
@@ -1,65 +1,67 @@
-{ cmake
-, fetchFromGitHub
-, lib
-, llvmPackages_17
-, cubeb
-, curl
-, extra-cmake-modules
-, ffmpeg
-, libaio
-, libbacktrace
-, libpcap
-, libwebp
-, libXrandr
-, libzip
-, lz4
-, makeWrapper
-, pkg-config
-, qtbase
-, qtsvg
-, qttools
-, qtwayland
-, SDL2
-, soundtouch
-, strip-nondeterminism
-, vulkan-headers
-, vulkan-loader
-, wayland
-, wrapQtAppsHook
-, xz
-, zip
-, zstd
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  llvmPackages_17,
+  cubeb,
+  curl,
+  extra-cmake-modules,
+  ffmpeg,
+  libaio,
+  libbacktrace,
+  libpcap,
+  libwebp,
+  libXrandr,
+  libzip,
+  lz4,
+  makeWrapper,
+  pkg-config,
+  qt6Packages,
+  SDL2,
+  shaderc,
+  soundtouch,
+  strip-nondeterminism,
+  vulkan-headers,
+  vulkan-loader,
+  wayland,
+  zip,
+  zstd,
 }:
 
 let
   # The pre-zipped files in releases don't have a versioned link, we need to zip them ourselves
   pcsx2_patches = fetchFromGitHub {
     owner = "PCSX2";
     repo = "pcsx2_patches";
-    rev = "e3b354f144de71d2b87471166cca8911867c1dfd";
-    sha256 = "sha256-H7cFyBYZumcCZ0/FFOFZoChoi0XPs4siA4dHcFt9U7k=";
+    rev = "b3a788e16ea12efac006cbbe1ece45b6b9b34326";
+    sha256 = "sha256-Uvpz2Gpj533Sr6wLruubZxssoXefQDey8GHIDKWhW3s=";
   };
+  inherit (qt6Packages)
+    qtbase
+    qtsvg
+    qttools
+    qtwayland
+    wrapQtAppsHook
+    ;
```

Comment:
qt6Packages.callPackages should be used instead of inheriting here, like it was done before.

Reply:
Why?

Reply:
Same reason as in the other PR. pkgs-by-name is not meant to replace existing wrappers with more copy paste code and for those it is totally fine to keep the current way.